utils/dbz_event_to_fhir_config.json

```diff
@@ -1,4 +1,18 @@
 {
+  "dbzConfigs": {
```

Comment:
try keeping the same complete naming style as eventConfigurations to keep things uniform.